packages/components/breadcrumb/README.md

```diff
@@ -11,14 +11,14 @@ To discuss or contribute to this component, visit the [GitHub issue for this com
 ### HTML markup
 
 ```html
-<nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+<nav class="nhsuk-breadcrumb" aria-label="Breadcrumb" >
```

Comment:
Would you be able to look at the breadcrumb njk file remove the space between the`"` and closing tag `>`